examples/compose/compose.yml

```
@@ -2,10 +2,10 @@ name: addy
 
 services:
   db:
-    image: mariadb:10.5
+    image: mariadb:11.3.2
```

no need to pin to patch release

```suggestion
    image: mariadb:11.3
```